index.js

```diff
@@ -44,11 +44,14 @@ const utils = require('rollup-pluginutils')
 
 module.exports = function rollupPluginPosthtml (options) {
   options = Object.assign({
+    emitFile: true,
```

Comment:
It will be very good if you add a description of this option in the readmi

Reply:
Done

Reply:
I think it should default to `false` and save the current behavior, thus no breaking change bump.